packages/sourcecred/src/api/main/grain.js

```diff
@@ -67,6 +68,19 @@ export function executeGrainIntegrationsFromGrainInput(
   const integrationCurrency = grainInput.currencyDetails.integrationCurrency;
   const grainIntegration = grainInput.grainConfig.integration;
   const results = [];
+  if (!ledger.accounting().enabled) {
+    const ledgerCurrency = ledger.accounting().currency;
+    if (!deepEqual(ledgerCurrency, integrationCurrency)) {
+      integrationCurrency
+        ? ledger.setExternalCurrency(
+            integrationCurrency.chainId,
+            integrationCurrency.tokenAddress
+              ? integrationCurrency.tokenAddress
+              : undefined
+          )
+        : ledger.removeExternalCurrency();
```

Comment:
I realized this evening that I need to do this for `enable/disableAccounting` and `enable/disableDistributionTracking` as well.